components/layout/header/nav-bar.tsx

```diff
@@ -16,7 +16,7 @@ interface Props {
 const NavBar = ({ showSearch }: Props) => (
   <Header className="header">
     <a className="logo" href="/">
-      <GameCiLogo height="30" />
+      <GameCiLogo width="60" height="60" />
```

Comment:
Width should be automatic based on the height.

Reply:
I tried it and for some reason, the logo gets super large when only height is specified (weird);
![image](https://user-images.githubusercontent.com/1264761/116820952-97a1f980-ab45-11eb-87db-017f66eba373.png)

But using only `width` instead of only `height` does work;

 
![image](https://user-images.githubusercontent.com/1264761/116820976-b56f5e80-ab45-11eb-850c-be6f63af3c9b.png)

so I'll go with `width` only.

Reply:
That's weird, when I tried height only (in dev tools) it worked just fine.

Either way, not setting height can break the menu bar (especially when a logo changes aspect ratio), so height definitely needs to be set.

Reply:
Used both in the end, but we can try again later. For now, navbar height doesn't seem to change depending on the screen width, but it probably has something to do with a css class or something like that.

Anyway, it looks better with logo than without. We can investigate this later, it's quite minor :)